src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
```

Comment:
The `SupplyOperation` class should be placed in the `strategy` package as per the checklist requirement to gather classes with the same purpose/common logic in a corresponding package.